Resources/Prototypes/Wires/layouts.yml

```diff
@@ -21,6 +21,23 @@
   parent: Airlock
   id: AirlockArmory
 
+- type: wireLayout
+  id: HighSec
+  wires:  
+  - !type:PowerWireAction
+    pulseTimeout: 10
+  - !type:PowerWireAction
+    pulseTimeout: 10
+  - !type:PowerWireAction
+    pulseTimeout: 10
+  - !type:PowerWireAction
+    pulseTimeout: 10
```

Comment:
why 4 power wires? doesnt that make it less secure than the normal doors

Reply:
no because it requires more precise cutting lest it get bolted.